src/Simulation/qdk_sim_rs/prerequisites.ps1

```diff
@@ -22,5 +22,5 @@ if (-not (Get-Command rustup -ErrorAction SilentlyContinue)) {
 # rustfmt and clippy is available.
 rustup install nightly
 rustup toolchain install nightly
-rustup component add rustfmt clippy
-rustup component add rustfmt clippy --toolchain nightly
+rustup component add rustfmt clippy llvm-tools-preview
```

Comment:
Are these needed by `qdk_sim_rs` as well now?

Reply:
no, but they are needed by the workspace now. So if someone runs even a simple `cargo check` in a folder other than src/Simulation/qdk_sim_rs it will also try to use llvm-tools-preview component for the build.rs in the qir_stdlib. For that reason I thought it made sense to include them here too.